lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -822,6 +830,7 @@ LowererMDArch::LowerCall(IR::Instr * callInstr, uint32 argCount)
 
         if (callInstr->GetSrc1()->IsHelperCallOpnd())
         {
+            callInstr->m_func->SetHasJitCalls();
```

Comment:
> callInstr->m_func->SetHasJitCalls(); [](start = 11, length = 37)

You should move this to Lower.cpp. You can check for the CALL opcode and call the setter. This way, you can avoid duplicated code across architectures.